docs/spec/v1beta2/imageupdateautomations.md

```diff
@@ -192,6 +192,46 @@ tuned to adjust the Git operation timeout.
 The proxy configurations are also derived from the referenced GitRepository
 source. `GitRepository.spec.proxySecretRef` can be used to configure proxy use.
 
+`GitRepository` can be configured to specify an OIDC
+[provider](https://github.com/dipti-pai/source-controller/blob/git-azure-oidc-auth/docs/spec/v1/gitrepositories.md#provider)
```

Comment:
TODO: replace the doc link here and in pre-requisites from fluxcd/source-controller once PR https://github.com/fluxcd/source-controller/pull/1591 is merged to SC repo.